### ISSUE TITLE
Front page to be more visual

### ISSUE THREAD

**Issue by mary-boti** [code-mentoring/code-mentoring.github.io#29]
1. I would do the front page more visual. I like the initial draft of the journey map presented in the bottom (which starts with "Intro in JS"). I would work on improving it and adding it up in the top. We have to catch ppl's attention and show them in few seconds what are the benefit of being with us. Visual is always great for this!
~2. I would brainstorm on slogan as well. It's great that we have it (love it!!). Just thinking that "A community that happens to code." may be not the strongest version of what we do. I personally not 'happens' to code hahah I do it for a purpose lol~ (moved to #35)

**Comment by mary-boti** [comment]
~I would say, in the front page we have to tell ppl on WHY they should be with us. After that guide them on HOW (via meet-up for the live session, via projects for challenges and improvements, via recommended learning platforms for growth)~ (moved to #35)

**Comment by edgul** [comment]
This is two issues, I will split into such.

**Comment by edgul** [comment]
This is a good idea. We just need to make it more concrete. How exactly can we make the page more visual? FYI we have access to assets (images) for the codementoring.co site

**Comment by mary-boti** [comment]
Good question Ed, hahah very good question!

Short answer - I see it being **visualized with a journey map** (kinda from the fist line of code to advanced). Fingers crossed we can find easy way to create the image. Let's first agree on the idea, and then to find the technical solution to it. (We may then use assets (like logo) to add into image.

Another option may be animated video (kinda Doodle). It's easier to create, content may be a pain - but actually I am not sure that from conceptual perspective it fits codementoring platform. 

*Just a few general examples
![Ex Journey Map 1](https://user-images.githubusercontent.com/77542735/107257176-4f85a600-6a32-11eb-96a5-c48dfa112377.jpg)
![Ex Journey Map 2](https://user-images.githubusercontent.com/77542735/107257180-50b6d300-6a32-11eb-8302-5085dbe82b53.jpg)
![Ex Journey Map 3](https://user-images.githubusercontent.com/77542735/107257182-50b6d300-6a32-11eb-968c-b10f117f5403.png)

**Comment by edgul** [comment]
I like the developer path idea, just not sure it needs to be our homepage. Perhaps there should be a dedicated page for developer paths.
As for the homepage, maybe it could be a unique image of something that demonstrates the importance of community.

**Comment by mary-boti** [comment]
That's true, it is too much info for the front page .... hmmm let me think of catchy front image ... agree it should be connected with the community

**Comment by ghost** [comment]
Importance of community would be good.

Developer journeys might come under FAQ, like "where do entry level developers start?", "what are some career paths?"

**Comment by mary-boti** [comment]
Great point Shane!

Added above FAQ into the G-doc. Pls feel free to add more suggestions :)
https://docs.google.com/spreadsheets/d/1hEmzQ5mxT9UJtri4ZB8ztJcr_rIiVHftW_LkwtoFGbY/edit#gid=0
